lib/Config.js

```diff
@@ -237,6 +237,60 @@ function hdClientLocationConstraintAssert(configHd) {
     return hdclientFields;
 }
 
+function azureArchiveLocationConstraintAssert(locationObj) {
```

Comment:
please add unit tests for this piece of code

Reply:
Done: https://github.com/scality/cloudserver/pull/5008/commits/95c50953064d4f0a5c4e79a4c4bde212aa342b04